packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -349,8 +350,36 @@ class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
                 ),
               ),
               const SizedBox(width: denseSpacing),
+              PopupMenuButton<_ScriptPopupMenuOptions>(
```

Comment:
nit: this build method is getting pretty long. can you break these items into two helpers `_buildScriptOptionsPopupMenu` and `_buildScriptHistoryPopupMenu`

Reply:
better yet, two stateless widgets